src/extension/background-script/connectors/connector.interface.ts

```diff
@@ -31,6 +62,12 @@ export type GetBalanceResponse = {
   };
 };
 
+export type GetInvoicesResponse = {
+  data: {
+    invoices: Invoice[];
```

Comment:
the response here should be a minimal(?) set of data that can be returned by all connectors. 

I actually would use a different type here for this. I think the connector interfaces ideally do not know anything about the rest of the alby application.